__tests__/integration-tests.test.js

```diff
@@ -237,6 +237,47 @@ describe('POST /api/articles/:article_id/comments test', () => {
     });
 });
 
+describe('DELETE /api/comments/:comment_id tests', () => {
+    test('should respond with only 204 with valid comment_id. Comments table should have the relevant entry removed', () => {
+        request(app)
+        .get("/api/articles/9/comments")
+        .then(({ body }) => {
+            expect(body.comments[0].comment_id).toBe(1)     
+            expect(body.comments[1].comment_id).toBe(17)     
+            expect(body.comments).toHaveLength(2)     
+        })
+        return request(app) 
+        .delete("/api/comments/1")
+        .expect(204)
+        .then((response) => {
+            expect(response.body).toMatchObject({})
+            return request(app)
+            .get("/api/articles/9/comments")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.comments).toHaveLength(1)
+                expect(body.comments[0].comment_id).toBe(17)     
+            }) 
+        })
+    });
```

Comment:
This is an interesting way to test this, but there are a couple of things that are a little confusing. I am not sure why there is an assertion checking for the comment_id to be 17, and the assertion to check that the body is an empty object for the delete endpoint is not needed as you cant send back anything so this will never fail. Overall its an interesting way to test that a comment has successfully deleted.